ui/src/message_popup/popup_question.jsx

```diff
@@ -118,19 +154,29 @@ export default React.createClass({
           student={student}
           question={question}
         />
-        {showResponse && <ResponseRenderer
-          scaffolding={scaffolding}
-          question={question}
-          limitMs={limitMs}
-          onResponseSubmitted={this.onResponseSubmitted}
-          onLogMessage={this.logData}
-        />}
+        {allowResponding && this.renderResponse()}
```

Comment:
Yeah thanks for pointing this out!  I noticed this earlier but it went away.  Poking around now but no luck yet, so I'll maybe spend a little more time and if not workaround for now by forcing it to be a particular size with styles or something slightly hacky like that.